applications/mne_anonymize/mne_anonymize.pro

```diff
@@ -40,15 +40,41 @@ include(../../mne-cpp.pri)
 
 TEMPLATE = app
 
-QT += widgets concurrent network
+QT += core gui widgets
```

Comment:
Removing the `network` module leads to linkage problems in windows static builds. 

Also, by default `Qt` includes `gui` and `core`. So no need to set them here again. You can safely use `QT += widgets network`

Reply:
will do